DogeNet.DAL/Models/ApplicationRole.cs

```diff
@@ -0,0 +1,12 @@
+﻿// <copyright file="DBContext.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.DAL.Models
+{
+    using Microsoft.AspNetCore.Identity;
+
+    public class ApplicationRole : IdentityRole<int>
```

Comment:
Why do we need an empty class?

Reply:
we need to create custom IdentityRole in case we want change string PK to int